docs/en_us/platform_api/source/courses/courses.rst

```diff
@@ -0,0 +1,274 @@
+Use Case
+=====================
+
+Get a list of the courses that are visible to a specific user. If the request
+is made by an anonymous user, a username is not required.
+
+=====================
+Example Requests
+=====================
+
+GET /api/courses/v1/courses/?&username=<username>
+
+GET /api/courses/v1/courses/
+
+=====================
+Query Parameters
+=====================
+
+* username (optional) - The username of the user for whom the course data is
+  being accessed. If the request is made by an anonymous user, username is not
+  required.
+
+* org (optional) - A code for an organization; case-insensitive. Example:
+  "HarvardX". If ``org`` is specified, the list of courses only includes
+  courses belonging to the specified organization.
+
+* mobile (optional) - If specified, the list of courses only includes courses
+  that are designated as ``mobile_available``.
+
+
+=====================
+Response Values
+=====================
+
+The following fields are returned with a successful response.
+
+* blocks_url: Used to fetch the course blocks.
+
+* effort: A textual description of the weekly hours of effort expected in the
+  course.
+
+* end: The date on which the course ends.
+
+* enrollment_end: The date on which enrollment ends.
+
+* enrollment_start: The date on which enrollment begins.
+
+* id: A unique identifier of the course; a serialized representation of the
+  opaque key identifying the course.
+
+* media: An object that contains named media items, including the following
+  objects.
+
+  * course_image: An image to show for the course.
+
+    * uri: The location of the image.
+
+  * course_video: A video about the course.
+
+    * uri: The location of the video.
+
+  * image: URLs for images in the course, including ``raw``, ``small``, or
+    ``large`` versions.
+
+* name: The name of the course.
+
+* number: The catalog number of the course.
+
+* org: The name of the organization that owns the course.
+
+* overview: An HTML textual description of the course.
+
+* short_description: A textual description of the course.
+
+* start: The date on which the course begins.
+
+* start_display: The start date of the course, formatted for reading.
+
+* start_type: Indicates how ``start_display`` was set. Possible values are:
+
+  * "string": Manually set.
+  * "timestamp": Generated from the ``start`` timestamp.
+  * "empty": The start date is not shown.
+
+* pacing: The type of pacing for this course. Possible values are
+  ``instructor`` and ``self``.
+
+* course_id: The course key. This field might be returned but is deprecated.
+  You should use ``id`` instead.
+
+
+==============================================================
+Example Response Showing The List of Courses Visible to a User
+==============================================================
+
+.. code-block:: json
+
+    {
+
+     "blocks_url":"/api/courses/v1/blocks/?course_id=edX%2Fexample%2F2012_Fall",
+     "media": {
+        "course_image": {
```

Comment:
Depending on @clintonb and @jcdyer's responses here, we will want to update the media field in this example.